### PR TITLE
MNEMONIC-535: Use assert function to handle test results and messages…

### PR DESCRIPTION
**Pull request by Chunyonghe**
… instead of println

### CODE REVIEW

**Comment by djkevincr** [comment]
+1

**Comment by bigdata-memory** [comment]
Thanks, but looks it didn't pass the building, it seems need to "import org.testng.Assert" in source file mnemonic-core/src/test/java/org/apache/mnemonic/PMDKVolatileMemAllocatorNGTest.java, thanks.

File: mnemonic-core/src/test/java/org/apache/mnemonic/PMDKVolatileMemAllocatorNGTest.java [hunk 1]
```diff
@@ -60,7 +60,7 @@ public void testMemChunk() {
     for (int idx = 1; idx <= 5; ++idx) {
       int size = randomGenerator.nextInt(1024 * 1024) + 1024 * 1024;
       mch = act.createChunk(size);
-      System.out.println(String.format("[Seq.%d] addr : %X", idx, size, mch.get()));
+      Assert.assertNotNull(mch, String.format("[Seq.%d] size: %d ", idx, size));
```
Comment (bigdata-memory) [inline mnemonic-core/src/test/java/org/apache/mnemonic/PMDKVolatileMemAllocatorNGTest.java]: Please try to add "import org.testng.Assert" in the head of this source file, thanks.

**Comment by Chunyonghe** [comment]
"import org.testng.Assert"  added in source file mnemonic-core/src/test/java/org/apache/mnemonic/PMDKVolatileMemAllocatorNGTest.java. And the issue resolved. Thanks!

**Comment by bigdata-memory** [comment]
Could you please squash 3 commits to one ? thanks.

**Comment by Chunyonghe** [comment]
Commits squashed completed. Thanks.

**Comment by bigdata-memory** [comment]
> Commits squashed completed. Thanks.

Thanks.

**Pull request merged**